packages/flutter/test/cupertino/text_field_test.dart

```diff
@@ -3446,7 +3446,7 @@ void main() {
       );
       await tester.tapAt(endpoints[0].point + const Offset(1.0, 1.0));
       await tester.pump();
-      await tester.pump(const Duration(milliseconds: 200)); // skip past the frame where the opacity is zero
+      await tester.pump(const Duration(milliseconds: 300)); // skip past the frame where the opacity is zero
```

Comment:
Is there a constant somewhere that could be used in all of the cases where you use 300ms to make it more clear where this comes from?

Reply:
Yeah I can do that 300ms is the kDoubleTapTimeout from gestures/constants.dart